hack/images

```diff
@@ -10,7 +10,9 @@ set -eu -o pipefail
 : "${RELEASE=false}"
 : "${PLATFORMS=}"
 : "${TARGET=}"
+: "${BUILDKITD_TAGS=}"
```

Comment:
I don't think these do anything as they are not used as vars in the script. Even if you would set default values to them in here that would not be the value that `--build-arg` would read from env.